### PR TITLE
LET'S GET SPOOKY

### PR DESCRIPTION
**Pull request by ghost**
I replaced the cult stuff with spooky skeleton stuff.
# I intend for this to be merged just before Halloween and subsequently reverted after the holiday is over.

### CODE REVIEW

**Comment by optimumtact** [comment]
http://www.oranges.net.nz/~oranges/webm/dootdoot.webm

**Comment by WJohn** [comment]
Don't we have a system in place that allows for pieces of code to be enabled on certain days of the year, specifically for holidays?

File: code/game/gamemodes/cult/runes.dm [hunk 1]
```diff
@@ -985,7 +986,7 @@ var/list/teleport_other_runes = list()
 	user << "<span class='warning'>Your blood begins flowing into [src]. You must remain in place and conscious to maintain the forms of those summoned. This will hurt you slowly but surely...</span>"
 	new_human.key = ghost_to_spawn.key
 	ticker.mode.add_cultist(new_human.mind)
-	new_human << "<span class='purple'><b><i>You are a servant of the Geometer. You have been made semi-corporeal by the cult, and you are to serve them at all costs.</i></b></span>"
+	new_human << "<span class='purple'><b><i>You are a servant of the Trumpted One. You have been made semi-corporeal by the cult, and you are to serve them at all costs.</i></b></span>"
```
Comment (MrPerson) [inline code/game/gamemodes/cult/runes.dm]: Trumpted One

**Comment by MrPerson** [comment]
I made the fucking holidays datums JUST FOR SHIT LIKE THIS!!! Fucking use them

**Comment by Fox-McCloud** [comment]
This should be something that you can enable any time of the year....ideally with custom sprites...

**Comment by Xhuis** [comment]
we have holiday datums?
fuck

**Pull request closed without merging**